modules/transport-netty4/src/internalClusterTest/java/org/opensearch/transport/netty4/Netty4HttpChannelsReleaseIntegTests.java

```diff
@@ -66,8 +67,17 @@ public void testAcceptedChannelsGetCleanedUpOnTheNodeShutdown() throws Interrupt
         }
         countDownLatch.await();
 
-        // no channels get closed in this test, hence we expect as many channels as we created in the map
-        assertEquals("All channels remain open", initialHttpChannels + numChannels, RestCancellableNodeClient.getNumChannels());
+        // no channels get closed in this test, hence we expect as many channels as we created in the map at most
+        // it is difficult to match the exact number of HTTP channels since:
+        // - there is 10 connections per route default setting (RestClient)
+        // - there are at least 2 nodes in the cluster (master + data)
+        // - if additional plugins are used (like telemetry), the same HTTP channel may
+        // be wrapped around, inflating the number of HTTP channels being tracked
+        assertThat(
+            "All channels remain open",
+            RestCancellableNodeClient.getNumChannels(),
+            greaterThanOrEqualTo(initialHttpChannels + 10 /* default connections per route */)
```

Comment:
@fdesu wondering what do you think, I believe it is sufficient to make sure that we have more channels than before we started the test, the exact number may vary indeed, thank you.